### PR TITLE
Rename role to label

### PR DESCRIPTION
**Pull request by raminqaf**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-kpops~=3.0
+kpops~=7.0
```
Comment (disrupted) [inline requirements.txt:1]: forgot couple releases 😄
todo: 8.0

Comment (raminqaf) [reply requirements.txt]: Bumped to 8.0

**Review by disrupted — COMMENTED** [review]
LGTM, but do we need to rebuild the apps against streams-bootstrap v3?

**Comment by raminqaf** [comment]
> LGTM, but do we need to rebuild the apps against streams-bootstrap v3?

Yes we do. But renaming the role to label is only kpops intern. 
I will make a follow-up PR and change the types to stream-app-v2 and producer-app-v2.

**Pull request merged**